src/instructlab/model/train.py

```diff
@@ -281,6 +294,54 @@
     default=False,
     help="if true, enables the legacy linux training codepath from release 0.17.0 and prior.",
 )
+@click.option(
+    "--phased-training",
+    is_flag=True,
+    default=False,
```

Comment:
I believe False is implied since we use `is_flag`. This can be removed.

Reply:
done!